server/src/server.ts

```diff
@@ -255,6 +259,14 @@ async function validatePerlDocument(textDocument: TextDocument): Promise<void> {
         nLog("Perl Critic Time: " + (Date.now() - start)/1000 + " seconds", settings);
         sendDiags({ uri: textDocument.uri, diagnostics: allNewDiags });
     }
+
+    const diagImports = await pImports;
+    documentDiags.set(textDocument.uri, diagImports); // May need to clear out old ones if a user changed their settings.
+    if(settings.perlimportsTidyEnabled){
+        const allNewDiags = perlOut.diags.concat(diagImports);
+        nLog("perlimports Time: " + (Date.now() - start)/1000 + " seconds", settings);
```

Comment:
I feel like a template string here would look prettier:
```suggestion
        nLog(`perlimports Time: ${Date.now() - start) / 1000} seconds`, settings);
```

Reply:
That said, I just noticed there is existing code above that also uses concatenation so maybe u want to keep it consistent